### ISSUE TITLE
Update default whiteboard colour to a brighter colour

### ISSUE THREAD

**Issue by tylercopeland** [bigbluebutton/bigbluebutton#5691]
I've been using the whiteboard feature in the HTML5 client a lot lately to review design projects with @ffdixon and almost everytime I find myself switching the colour to something that's brighter (red, pink or bright blue.) 

I think it would be great if the HTML5 client had a bright colour selected by default. Either red or blue.

**Comment by oswaldoacauan** [comment]
Related to this we should probably add a colorpick option so people can choose custom colors and not only the preseted ones.

**Comment by antobinary** [comment]
Let's make the default color red. We can look into the colorpick in a different isue

**Comment by tylercopeland** [comment]
@antobinary that works for me!

**Comment by akiwenziec** [comment]
To be completely honest I thought the cursor already selected as red was a bug. 

Do we have any customer feedback regarding this or that was used to choose this particular colour as the default choice?  

Not sure this is necessary for the end user  - may result in requests to have this default colour selected customized at the account level (not sure that can be done)

**Comment by tylercopeland** [comment]
@akiwenziec great questions and thinking here. 

My initial thoughts were to provide the instructor (by default) with a colour that is most likely different than the average colour found on a slide. Something that would stand out during and after the presentation has ended. Today our default colour is black. 

I did a bit of research of existing annotation tools in the market, and it seems like a common practice is to use red as the default colour, or yellow (highlight). Even simple searches of "annotation" bring back images of people using red to annotate items on a slide or document.

But, on the flip side, there is a market of BBB users who use the tool primarily as a whiteboarding solution to teach subjects such as Math. In that particular use case, I agree, red might not be the best default colour. Furthermore and to your point, existing users of BigBlueButton (flash) aren't familiar with seeing red as the default colour which would lead to further support requests.

Regarding, colour blindness and accessibility. The current red used does not meet the accessibility contrast ratio, which is a great catch. Thank you.

**Current colour red used**

![screen shot 2019-01-17 at 12 28 57 pm](https://user-images.githubusercontent.com/1273169/51336834-8d7e8e80-1a53-11e9-9f46-6e077bcd8bce.png)

I will open a new ticket to make sure all our annotation colours pass accessiblity contrast ratios. 

In summary, I'm happy to leave the the default colour as black if we all think it's best for our users/use cases and return to this idea only until our users ask for it. 

@antobinary @ffdixon thoughts?

**Comment by akiwenziec** [comment]
@tyler 

Right now when the HMTL5 client loads the pencil tool is the default tool selected with a default colour:


![screen shot 2019-01-17 at 1 48 59 pm](https://user-images.githubusercontent.com/18176998/51341606-b9077600-1a5f-11e9-9980-454b79561f01.png)

May I offer the following suggestions
-making the default tool the "Pan" or hand tool with the cursor showing as red to make it more visible. 

![screen shot 2019-01-17 at 1 49 20 pm](https://user-images.githubusercontent.com/18176998/51341643-cf153680-1a5f-11e9-9be1-14b0b864faa8.png)


I agree the viewer should be able to immediately see the cursor or "dot" to identify the content the moderator/presenter is calling attention to and the current Flash client's default cursor/dot is red, which we have not had any issues or complaints regarding (however we can account for this now in the mobile client) 

![screen shot 2019-01-17 at 1 56 22 pm](https://user-images.githubusercontent.com/18176998/51341662-d9373500-1a5f-11e9-9b4d-d421e30ad2a4.png)


-when selecting the pencil tool the default colour can be a brighter colour such as yellow or orange orange may be better to avoid issues with yellow/blue spectrum colourblindness (which is much less common than red/green spectrum).

**Comment by roulili** [comment]
Hi all!
Is there any chance that we can change this default colour in the client settings.yml ?

We're using BBB exactly as @tylercopeland described, mainly for the whiteboarding capacity, and red is not always well perceived by our userbase.

**Comment by matiasilva** [comment]
Seeing as the original issue this issue addressed has been merged and solved, could you open a new issue for your feature enhancement? I'll close this one now.
